dandischema/metadata.py

```diff
@@ -168,7 +195,7 @@ def validate(obj, schema_version=None, schema_key=None, missing_ok=False):
                 reraise = True
                 messages.append(el["msg"])
         if reraise:
-            ValueError(messages)
+            raise ValueError(messages)
```

Comment:
and that would be another location to use the aforementioned encapsulation of errors.